frontend/src/pages/CompanyAdmin.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Route } from 'react-router-dom'
+import FullProductList from '../components/FullProductList'
+
+const CompanyAdmin = () => {
```

Comment:
Ezt hol használjuk?

Reply:
Egyelőre még sehol, törlöm a kódból, hogy a hozzá tartozó issue-val kerüljön csak be.